internal/lightcone/preservation/wearewildfire/wearewildfire.go

```diff
@@ -0,0 +1,58 @@
+package wearewildfire
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	mod     key.Modifier = "wearewildfire"
+	modHeal key.Modifier = "wearewildfire-heal"
+)
+
+// At the start of the battle, the DMG dealt to all allies decreases by
+// 8%/10%/12%/14%/16% for 5 turn(s). At the same time, immediately restores HP
+// to all allies equal to 30%/35%/40%/45%/50% of the respective HP difference
+// between the characters' Max HP and current HP.
+func init() {
+	lightcone.Register(key.WeAreWildfire, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_PRESERVATION,
+		Promotions:    promotions,
+	})
+}
+
+func Create(engine engine.Engine, owner key.TargetID, lc info.LightCone) {
+	// team DMG RES
+	amt := 0.06 + 0.02*float64(lc.Imposition)
+	amtHeal := 0.25 + 0.05*float64(lc.Imposition)
+
+	dmgmod := info.Modifier{
+		Name:     mod,
+		Source:   owner,
+		Stats:    info.PropMap{prop.AllDamageReduce: amt},
+		Duration: 5,
+	}
+
+	heal := info.Heal{
+		Targets: engine.Characters(),
+		Source:  owner,
+		BaseHeal: info.HealMap{
+			model.HealFormula_BY_TARGET_LOST_HP: amtHeal,
+		},
+	}
```

Comment:
Need to move this `info.Heal` construction into the `Subscribe` function (specifically the `engine.Characters()` call. In the current state, will get the list of characters here, before the `BattleStart` event occurs. Depending on which character has this LC, not all characters will be in `engine.Characters()` return (IE: if this is first char in the party order, this list will be empty since they are created first)